WordPress/Classes/Networking/BlogServiceRemote.h

```diff
@@ -4,9 +4,14 @@
 
 typedef void (^OptionsHandler)(NSDictionary *options);
 typedef void (^PostFormatsHandler)(NSDictionary *postFormats);
+typedef void (^MultiAuthorCheckHandler)(BOOL isMultiAuthor);
 
 @protocol BlogServiceRemote <NSObject>
 
+- (void)checkMultiAuthorForBlog:(Blog *)blog
+                        success:(MultiAuthorCheckHandler)success
+                        failure:(void (^)(NSError *error))failure;
```

Comment:
+1 to the both of you. It'd be cool if we began using typedef's, but i think the full code deserves a review, since this is the existing pattern all over.